Makefile

```diff
@@ -14,16 +14,25 @@ endif
 
 all: manager
 
+# Create a local docker registry and start kinD cluster
+start-kind-with-registry:
+	bash scripts/kind-with-registry.sh
+
+# Delete the local docker registry and the kinD cluster
+stop-kind-with-registry:
+	bash scripts/teardown-kind-with-registry.sh
+
 dev:
-	-kind create cluster --config .kind-cluster.yaml --name orkestra
+	docker build . -t localhost:5000/orkestra:dev
+	docker push localhost:5000/orkestra:dev
```

Comment:
We still want to create kind cluster here. 
What we are trying to do with this target is:
1. Create a local k8s cluster
2. Install orkestra **without** the orkestra controller running as part of the release
3. Use `debug` target to run the local instance (optional). You can also start it in debug mode yourself if you wish to use `dlv` debugger.

The docker build and push should be standalone targets used only if you wish to run them locally on your kind cluster. Might not be the most useful feature but is good for testing the builds.

If you look at the latest developer docs we no longer need to run the docker steps.

Reply:
So should I change `dev` target to:
```shell
dev: start-kind-with-registry
    helm upgrade --install orkestra chart/orkestra --wait --atomic -n orkestra --create-namespace --values ${CI_VALUES}
```
or should I remove `start-kind-with-registry` target and change `dev` target to:
```shell
dev:
    bash scripts/kind-with-registry.sh
    helm...
```

Reply:
Either is fine - you most likely will not use one without the other, so it might make sense to move it to `dev`